ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # eRecruiter API ChangeLog
 
+## 1.30.0
+- New portal setting to indicate if pagination in the Jobs list should be enabled and the page size

```

Comment:
suggestion: 
New portal settings to indicate if pagination should be enabled in the jobs list and how many items should be displayed.